src/content/blog/astro-320.mdx

```diff
@@ -0,0 +1,69 @@
+---
+title: "Astro 3.2: View Transitions improvements"
+description: "Astro 3.2 is out with several improvements to View Transitions, including the ability to control the history stack, and a JavaScript navigation API"
+publishDate: "September 28, 2023"
+authors:
+  - matthew
+  - martrapp
+coverImage: "/src/content/blog/_images/astro-320/blog-hero-3_2-release.jpg"
+socialImage: "/src/content/blog/_images/astro-290/og-image-2_9-release.jpg"
+lang: "en"
+---
+
+Astro 3.2 is out and several new improvments for [view transitions](https://docs.astro.build/en/guides/view-transitions/), 
+
+- **[History Replacement on Links](#history-replacement-on-links)**
+- **[JavaScript Navigation API](#javascript-navigation-api)**
+- **[Route announcer for screen readers](#route-announcer)**
```

Comment:
I think your list was fine, but you could consider going more use-case for the blog post, something like (and update any links accordingly):

```suggestion
- **[Control the browser history stack](#history-replacement-on-links)**
- **[Trigger page navigation via JavaScript](#javascript-navigation-api)**
- **[Route announcer for screen readers](#route-announcer)**
```